app/views/application_submissions/show.haml

```diff
@@ -96,6 +96,7 @@
       .form-group
         Interview occurred on
         = @interview.information
+    = render 'save_for_later', locals: { record: @record }
```

Comment:
A little nitpick is that this already exists elsewhere. You can, I think, just change the logic that makes it display.